src/ape/utils.py

```diff
@@ -83,6 +105,15 @@ def get_relative_path(target: Path, anchor: Path) -> Path:
 
 
 def get_package_version(obj: Any) -> str:
+    """
+    Gets package version of packages.
+
+    Args:
+        obj (Any): object to search inside for version.
```

Comment:
```suggestion
        obj: object to search inside for version.
```

I don't think we can include a type when its `Any`